qiskit_nature/second_q/hamiltonians/quadratic_hamiltonian.py

```diff
@@ -84,68 +133,53 @@ def __init__(
             ValueError: Hermitian part must be Hermitian.
             ValueError: Antisymmetric part must have shape num_modes x num_modes.
             ValueError: Antisymmetric part must be antisymmetric.
+
+        Note:
```

Comment:
Since the atol & rtol values above are defaulted to the same as the Mixin atol and rtol would it be preferable, like is done in other methods using the mixin where local atol and rtol can be passed for the parameters to default to None. Then if None the class level atol/rtol are used?

Reply:
Done.